CRM/Activity/Selector/Activity.php

```diff
@@ -130,7 +128,9 @@ public static function actionLinks(
 
       case 'Payment':
       case 'Refund':
-        $participantId = CRM_Core_DAO::getFieldValue('CRM_Event_BAO_ParticipantPayment', $sourceRecordId, 'participant_id', 'contribution_id');
+        if (!$sourceRecordId) {
```

Comment:
@eileenmcnaughton isn't this reversed from what it should be?

Reply:
erm yes....

Reply:
@seamuslee001 now you see it now you don't